docs/vsphere-golden-image.md

```diff
@@ -0,0 +1,172 @@
+
+The private key of [the secret created while installing WMCO](https://github.com/openshift/windows-machine-config-operator#usage) to give OpenShift Container Platform access to the Windows worker nodes. 
+
+The `authorized_keys` file is used to configure the OpenSSH Server in the Windows VM.
+
+The provided PowerShell script [install-openssh.ps1](vsphere_ci/scripts/install-openssh.ps1) aims to programatically 
+install and configure the OpenSSH Server with the given `authorized_keys` file.
+
+
+## 4. Setup the container runtime
+
+> Currently `docker`, it will change to `containerd`
+
+Install the `docker` container runtime following the [Microsoft's documentation](https://docs.microsoft.com/en-us/virtualization/windowscontainers/quick-start/set-up-environment?tabs=Windows-Server).
+
+In addition, you can run the provided PowerShell script [install-docker.ps1](vsphere_ci/scripts/install-docker.ps1) to programatically install `docker` in the Windows VM.
+
+
+### Pull required Windows container base images
+
+Pull all of the required Windows container base images needed for your applications. Such images are kernel-dependent
+on the Windows Server version you are using. See [Microsoft’s documentation](https://docs.microsoft.com/en-us/virtualization/windowscontainers/manage-containers/container-base-images) on pulling Windows container base images 
+for more information and to determine which images you need.
+
+To display the current Windows OS version using PowerShell
+
+```powershell
+    Get-ComputerInfo | Select OsHardwareAbstractionLayer 
+```
+
+Example base images:
+- Windows Server 1909 SAC: `docker pull mcr.microsoft.com/windows/servercore:1909`
+- Windows Server 2004 SAC: `docker pull mcr.microsoft.com/windows/servercore:2004`
+
+For additional Windows Server Core available images see the [Docker Hub repository](https://hub.docker.com/_/microsoft-windows-servercore?tab=description).
+
+
+## 5. Setup incoming connection for container logs
+
+Create a new firewall rule in the Windows VM to allow incoming connections for Container Logs, usually 
+on TCP port `10250` by running the following PowerShell command:
+
+```powershell
+    New-NetFirewallRule -DisplayName "ContainerLogsPort" -LocalPort 10250 -Enabled True -Direction Inbound -Protocol TCP -Action Allow -EdgeTraversalPolicy Allow
+```
+
+In addition, you can run the provided PowerShell script [install-firewall-rules.ps1](vsphere_ci/scripts/install-firewall-rules.ps1) to programatically create the container logs rule along with the OpenSSH incoming connections rule.
+
+
+## 6. Generalize the virtual machine installation
+
+To deploy the Windows VM as a reusable image, you have to first generalize the VM removing computer-specific information 
+such as installed drivers. The `Sysprep` with an *unattend* answer file generalize the image and make it ready for 
+deployment, maintaining all the changes needed for the WMCO installation. 
```

Comment:
Do you mean `Running sysprep with an unattend answer file generalizes the image and makes...`?